docs/DeveloperGuide.md

```diff
@@ -166,16 +166,17 @@ creates the connection from the ```Main``` class to the other classes required s
 
 To add a patient, the ```AddPatientCommand``` class is used. For this ```AddPatientCommand``` class, it serves as the 
 façade class for the ```Main```, ```Patient``` , ```PatientList``` and the ```Storage``` class to interact with one 
-another. 
+another. Also, to uniquely identify a patient, an unique patientId number is assigned to each patient when they are first added into the patient list.
 
 ![](images/AddPatientDiagram.png)
 
 1. The ```AddPatientCommand``` class object will first be created by the ```Parser``` object, where the information 
-regarding the patient to be added will be stored in the ```AddPatientCommand``` class object. 
+regarding the patient to be added will be stored in a Map, where the ```AddPatientCommand``` class object would read the Map content and store the information about the patient in said ```AddPatientCommand``` class object. 
+For the patient Id number, it will call upon the static class ```patientIdManager``` to get its unique patient id number. This unique Id number will be used later in the ```Patient``` object creation too.
 
 2. When the 
 ```execute(Ui ui, Storage storage)``` command is called, the  ```AddPatientCommand``` would first make use of the 
-```Patient``` class constructor to create a new ```Patient``` object. 
+```Patient``` class constructor to create a new ```Patient``` object based on the information stored back in step 1. 
```

Comment:
I am unable to find the PR for AddPatientCommandSequence diagram but here are some comments about the diagram:
1. If I am not mistaken, forward/invoke arrows should not be dotted lines, only return arrows are dotted.
2. the loop part of diagram is quite hard to read as the arrows are squeezed, consider expanding the diagram.

![Screenshot (77)](https://user-images.githubusercontent.com/42404655/77895098-dad9cb00-72a8-11ea-94ed-03993b8eb5c0.png)